src/docfx/lib/markdown/ExtractTitle.cs

```diff
@@ -14,27 +15,67 @@ namespace Microsoft.Docs.Build
 {
     internal static class ExtractTitle
     {
+        [ThreadStatic]
+        private static bool s_headingFound = false;
```

Comment:
seems s_headingFound can be removed?

Reply:
yes!

---
In reply to: [212194562](https://github.com/dotnet/docfx/pull/3266#discussion_r212194562) [](ancestors = 212194562)